contexts/SubscriptionContext/tests/Integration/UseCases/AddSubscription/AddSubscriptionUseCaseTest.php

```diff
@@ -141,19 +143,4 @@ public function testGivenDataThatNeedsToBeModerated_requestNotBeMailed() {
 		$useCase->addSubscription( $request );
 	}
 
-	public function testNewMailAddressFromSubscription(): void {
-		$useCase = new AddSubscriptionUseCase( $this->repo, $this->validator, $this->mailer );
-		$class = new ReflectionClass( AddSubscriptionUseCase::class );
-
-		$method = $class->getMethod( 'newMailAddressFromSubscription' );
-		$method->setAccessible( true );
-
-		$subscription = $this->createMock( Subscription::class );
```

Comment:
It's weird to mock a non-service. Though since this is a Doctrine ORM thing and not something more simple I can see why you did it.

I'm writing this comment now since I realized that this should not be the case: UCs should not bind to data access details. And indeed there already is a FIXME for this in SubscriptionRepository